src/main/resources/db/migration/V5__tilkjent_ytelse_v3.sql

```diff
@@ -0,0 +1,35 @@
+"Stygg syntax-feil som stopper databaseoppdatering"
+
+DROP TABLE andel_tilkjent_ytelse;
+DROP TABLE tilkjent_ytelse;
+
+CREATE SEQUENCE ytelse_lopenummer_seq INCREMENT BY 1000 START WITH 1000;
```

Comment:
Hvorfor `INCREMENT BY 1000` ?

Reply:
Fordi denne brukes som start-teller for utbetalingene på en tilkjenning. Det gir plass til 1000 utbetalinger per ytelse uten at utbetalingsoppdragene vil få duplikate nøkler.

Reply:
Rent logisk, hva er det vi bruker den til? Kunne det bare vært et løpenummer slik at min første er 1, din første er 2 og min neste er 3, eller er det noe logikk i dette løpenummeret som jeg ikke har forstått ❓

Reply:
Jeg tror det er et ønske/behov i Oppdrag at alle Utbetalingsoppdrag for en ytelse skal komme etter hverandre.

Reply:
Hva menes med etter hverende da? Slik jag tolker dette ønsker man att dette skal skje

Utbetaling maj
Person A utbetaling 1 seq 1000
Person B utbetaling 1 seq 2000

Utbetaling juni
Person A utbetaling 2 seq 1001
Person B utbetaling 2 seq 2002

Hvordan ønsker de att den skal øke?
- Globalt
- På individnivå

Reply:
@blommish Person A får utbetalingsoppdrag 1000, 1001, 1002 og person B får 2000, 2001, 2002. Det stemmer. Hvorfor logikken er sånn vet jeg ikke. Jeg har bare forsøkt å ivareta funksjonaliteten Jobi lagde.